Tella/Scenes/Uwazi/ViewModel/UwaziEntityViewModel.swift

```diff
@@ -34,5 +40,59 @@ class UwaziEntityViewModel: ObservableObject {
         requiredPrompts.forEach { prompt in
             prompt.showMandatoryError = prompt.value.stringValue.isEmpty
         }
+        
+        if(!requiredPrompts.isEmpty) {
+            submitEntity()
+        }
+    }
+    
+    private func submitEntity() {
+        // Extract entity data and metadata
+        let entityData = extractEntityDataAndMetadata()
+        
+        // Prepare server URL and cookie list
+        let serverURL = self.serverURL
+        let cookieList = ["connect.sid=" + self.accessToken]
```

Comment:
yeah, I think we can do that. But let's leave that for another PR because it would require more specific work for adding the table and refactoring the current code with the new table. So let's first merge the remaining PRs about entity and I will work in parallel adding this new table.